### PR TITLE
Partners revamp

### PR DESCRIPTION
**Pull request by BrieQiel**
- applied dynamic pages to partners
- redesigned partners page (last na redesign na to pramis 😭)

### CODE REVIEW

File: pages/partners/[partner].vue [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+            </v-window>
+          </v-col>
+        </v-row>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+import { ref, onMounted } from 'vue';
```
Comment (gabesantos21) [inline pages/partners/[partner].vue]: Kahit wala na to. Nuxt should be able to auto import this

File: pages/partners/[partner].vue [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+import { useRoute } from 'vue-router';
+import { partnersData } from '../../data/partners-data.ts';
+
+useHead({
+  title: 'DataStax',
```
Comment (gabesantos21) [inline pages/partners/[partner].vue]: use a dynamic text here. it will always be DataStax

**Review by gabesantos21 — COMMENTED** [review]
Check comments

File: pages/partners/[partner].vue (hunk 2 above)
Comment (gabesantos21) [reply pages/partners/[partner].vue]: image

placing it in mounted would be the wei

File: pages/partners/[partner].vue [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+    cover.value = partnerDetails.value.cover;
+    name.value = partnerDetails.value.name;
+    background.value = partnerDetails.value.background;
+    tagline.value = partnerDetails.value.tagline;
+    description.value = partnerDetails.value.description;
+    products.value = partnerDetails.value.products;
+  }
+  useHead({
+    title: name.value,
+  });
```
Comment (gabesantos21) [inline pages/partners/[partner].vue]: place UseHead inside the if condition.

Comment (BrieQiel) [reply pages/partners/[partner].vue]: noted, will also implement page not found error

Comment (gabesantos21) [reply pages/partners/[partner].vue]: kahit gayahin mo nalang muna yung ginawa ko. like this

else {
    throw createError({ statusCode: 404, statusMessage: 'Page not found', fatal: true });
  }

Comment (gabesantos21) [reply pages/partners/[partner].vue]: yung custom error after nalang magmerge yung na pr ko

**Pull request merged**